### DISCUSSION TITLE
Diatone mamba f405 mini mk4 and vtx power

### DISCUSSION THREAD

**Discussion by didichung** [iNavFlight/inav#9535] [Support]
hello,
my vtx is always turn off,because i dont know how power on vtx power switch with inav 6.1.1…
(it is work well whit betaflight,but i prefere inav:-)))
thank you very much

my fc is mamba f405 mk4 2022b, I already succesfully flashed the right firmware back and forth between BF and inav... the result is still the same... my dji o3 not powered with inav firmware...

**Comment by andyp14feb** [comment]
got exactly the same problem here ... , my vtx is AKK Race Ranger

**Comment by andyp14feb** [comment]
Hi I just found the solution (accidentally), it works for me

just go to "modes" tab and set User1 and User2 turned on simultaneously with 1 switch

![image](https://github.com/iNavFlight/inav/assets/38460621/3d6dcc20-d965-4ced-b817-bd0d5f714a3f)

**Comment by TheIceman306** [comment]
I have the same FC and I got it work in the MODES tab by adding range for USER2 to be the entire range for channel 5 (arm switch). That way the O3 powers up as soon as a battery is plugged in.
![Screenshot 2024-03-29 102549](https://github.com/iNavFlight/inav/assets/139028187/63a5fe8b-f36f-4c50-84fa-5d73c3bb4c31)
